e2e/keywords/common.resource

```diff
@@ -17,22 +19,16 @@ Set test environment
     init_k8s_api_client
     init_node_exec    ${SUITE NAME.rsplit('.')[1]}
     init_storageclasses
-    @{volume_list} =    Create List
-    Set Test Variable    ${volume_list}
-    @{deployment_list} =    Create List
-    Set Test Variable    ${deployment_list}
-    @{statefulset_list} =    Create List
-    Set Test Variable    ${statefulset_list}
-    @{persistentvolumeclaim_list} =   Create List
-    Set Test Variable    ${persistentvolumeclaim_list}
+
     setup_control_plane_network_latency
 
 Cleanup test resources
```

Comment:
`Cleanup test resources` appears to be unable to execute concurrently if we intend to run tests concurrently. 

Currently, it's OK, but later on, for concurrent test execution, we should specifically tear down the resources for the tests that are currently running.